weblate/utils/tasks.py

```diff
@@ -82,16 +80,22 @@ def database_backup():
         cmd += ["--port", database["PORT"]]
     if database["USER"]:
         cmd += ["--username", database["USER"]]
+    if settings.BACKUP_COMPRESSED:
+        cmd += ["--file", data_dir("backups", "database.sql.gz")]
+        cmd += ["--format", "custom"]
+    else:
+        cmd += ["--file", data_dir("backups", "database.sql")]
 
     subprocess.check_call(cmd, env=get_clean_env({"PGPASSWORD": database["PASSWORD"]}))
 
 
 @app.on_after_finalize.connect
 def setup_periodic_tasks(sender, **kwargs):
-    sender.add_periodic_task(
-        crontab(hour=1, minute=0), settings_backup.s(), name="settings-backup"
-    )
-    sender.add_periodic_task(
-        crontab(hour=1, minute=30), database_backup.s(), name="database-backup"
-    )
+    if not settings.BACKUP_SKIPPED:
```

Comment:
Why disabling settings backup as well?

Reply:
Because at first I was doing the skip for the entire backup, but reading back your comments and the issue it should be indeed only for the DB, I'll change that.